tinyagentos/feedback_store.py

```
@@ -0,0 +1,118 @@
+        app TEXT NOT NULL DEFAULT '',
+        created_at TEXT NOT NULL
+    );
+    CREATE INDEX IF NOT EXISTS feedback_user_created
+        ON feedback (user_id, created_at DESC);
+    """
+
+    async def create(
+        self,
+        *,
+        user_id: str,
+        type: str,
```

Ruff (0.15.17)

[error] Function argument `type` is shadowing a Python builtin

(A002)